hat/assets/js/apps/Iaso/domains/users/components/ExportMobileAppSetupDialog.tsx

```diff
@@ -0,0 +1,265 @@
+/* eslint-disable camelcase */
+import React, {
+    FunctionComponent,
+    useCallback,
+    useEffect,
+    useMemo,
+    useState,
+} from 'react';
+import { MutateFunction } from 'react-query';
+import { Alert, Box, Button, Typography } from '@mui/material';
+import LinearProgress, {
+    LinearProgressProps,
+} from '@mui/material/LinearProgress';
+import {
+    ConfirmCancelModal,
+    IconButton,
+    IntlMessage,
+    makeFullModal,
+    useSafeIntl,
+} from 'bluesquare-components';
+
+import MESSAGES from '../messages';
+import InputComponent from 'Iaso/components/forms/InputComponent.tsx';
+import { useTaskMonitor } from 'Iaso/hooks/taskMonitor';
+import { getRequest } from 'Iaso/libs/Api';
+import { TaskApiResponse } from 'Iaso/domains/tasks/types';
+import { Project, User } from 'Iaso/utils/usersUtils';
+
+const styles: SxStyles = {
+    alert: {
+        marginBottom: '1rem',
+        marginTop: '1rem',
+    },
+    progress: {
+        width: '100%',
+    },
+    downloadBtn: {
+        display: 'flex',
+        justifyContent: 'center',
+        marginTop: '1rem',
+    },
+};
+
+type Props = {
+    titleMessage: IntlMessage;
+    selectedUser: User;
+    onCreateExport: MutateFunction<any>;
+    isOpen: boolean;
+    closeDialog: () => void;
+};
+
+function LinearProgressWithLabel(
+    props: LinearProgressProps & { value: number },
+) {
+    return (
+        <Box sx={{ display: 'flex', alignItems: 'center' }}>
```

Comment:
this is a FunctionComponent

Reply:
also you can use proper sxStyles for it

Reply:
> this is a FunctionComponent

Sorry, but what do you mean by this comment? Not sure what I should change